course-v2/assets/js/course_expander.ts

```diff
@@ -40,4 +40,22 @@ const initCourseInfoExpander = document => {
   }
 }
 
-export { initCourseInfoExpander }
+const initCourseDescriptionExpander = document => {
+  const courseDescription = document.getElementById("course-description")
+  const collapsedDescription = courseDescription.querySelector("#collapsed-description")
+  const expandedDescription = courseDescription?.querySelector("#expanded-description")
```

Comment:
Are we safe to assume `courseDescription` always exists?

I stand by the "ditch jquery" suggestion, but I had forgotten (until workin on #887) that the jquery versions are a little more forgiving if things don't exist.

Also: ... Typescript should have caught this issue, except there's a `ts-nocheck` at the top of the file. Not necessarily requesting you change it as part of this PR, but we should get rid of it.